pixel.py

```
@@ -0,0 +1,28 @@
+#!/usr/bin/python3
+
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)